internal/querycoord/query_coord.go

```diff
@@ -463,13 +470,14 @@ func (qc *QueryCoord) loadBalanceNodeLoop(ctx context.Context) {
 				meta:               qc.meta,
 			}
 			qc.metricsCacheManager.InvalidateSystemInfoMetrics()
-			//TODO:: deal enqueue error
+
 			err := qc.scheduler.Enqueue(loadBalanceTask)
 			if err != nil {
 				log.Warn("failed to enqueue LoadBalance task into the scheduler",
 					zap.Int64("nodeID", node),
 					zap.Error(err))
 				qc.offlineNodesChan <- node
+				time.Sleep(LoadBalanceRetryAfter)
```

Comment:
NIT: sleep or context.Done(), maybe use existing `waitWithContext` util func.

Reply:
This line is in a loop that select ctx.Done()